for-developers.md

```diff
@@ -0,0 +1,275 @@
+# Developer Guide
+
+1. [Configuration Settings](#configuration-settings)
+
+2. [Batch environment setup and fre-cli](#batch-environment-setup-and-fre-cli)
+
+3. [Post-processing guide using fre-cli and cylc](#guide)
+
+# Configuration Settings
+
+`fre-workflows` has primarily been developed and tested on PPAN. The following are guidelines for developing on PPAN:
+
+## Cylc User settings
+
+The path to the Cylc binary must be added to your `$PATH` environment variable first. This can be done by modifying your PATH in your bash_profile as follows:
+
+```
+> cat /home/First.Last/.bash_profile
+# this is essential Cylc configuration below
+export PATH="${PATH}:/home/fms/local/opt/cylc/bin"
+```
+
+We need this to run on PPAN. Without a cylc binary in your `$PATH` variable, the initial setup of the workflow server (check terminology) fails, and we cannot edit the environment with a module load or similar until it is running.
+
+> ** _NOTE:_** Adding a link to the cylc binary in a directory that is default in user's $PATH is desired, this is being persued.
+
+
+## Experiment settings
+
+The current organization of the yaml files comprises of the model, settings, and post-processing configuration files, where the model yaml holds paths that point to the settings and post-processing yamls.
```

Comment:
Currently, there are three different types of yaml files:   model, settings, and post-processing configuration yaml files.  
* the model yaml contains paths to the settings and post-processing yaml files

and then continue on with below.  

I think the "for more information" can be at the end of this section